chain-cli/chaincode-template/src/token/GalaChainTokenContract.ts

```diff
@@ -91,6 +91,8 @@ import { Info } from "fabric-contract-api";
 
 import { version } from "../../package.json";
 
+const curatorOrgMsp = process.env.CURATOR_ORG_MSP ?? curatorOrgMsp;
```

Comment:
I believe that was the intention:

```suggestion
const curatorOrgMsp = process.env.CURATOR_ORG_MSP ?? "CuratorOrg";
```